ext/intl/bidi/bidi.c

```diff
@@ -777,6 +855,7 @@ static PHP_METHOD(IntlBidi, getReordered) {
 		return;
 	}
 
+	// HERE IT CRASHES WHEN RUNNING IntlBidi_getReordered_varian.phpt (not enough memory allocated for the string).
```

Comment:
Hrmm, I'll take a look and see if anything obvious pops up.. :/

Reply:
Found a fix for it.
It had something to do with a reordering option of bidi (INSERT_MARKS).